modules/SitecoreImageBuilder/SitecoreImageBuilder.psm1

```diff
@@ -107,6 +107,11 @@ function Invoke-PackageRestore
                     {
                         throw ("Unable to login to '{0}' with the supplied credentials." -f $sitecoreDownloadUrl)
                     }
+                    else
+                    {
+                        # Assign the downloadSession variable back to sitecoreDownloadSession to use in package download - Line 120
+                        $sitecoreDownloadSession = $downloadSession 
```

Comment:
You can just delete this `else` completely and fix by changing the `-SessionVariable "downloadSession"` to `-SessionVariable "sitecoreDownloadSession"` in:

```
   $loginResponse = Invoke-WebRequest "https://dev.sitecore.net/api/authorization" -Method Post -Body @{
                        username   = $SitecoreUsername
                        password   = $SitecorePassword
                        rememberMe = $true
                    } -SessionVariable "downloadSession" -UseBasicParsing
```

I see that it was my rename of the variable that broke it, thanks for finding it!

Reply:
@pbering, I have updated the file as per review comments.